src/draw/vg_lite/lv_draw_vg_lite_vector.c

```diff
@@ -336,6 +343,43 @@ static void lv_path_to_vg(lv_vg_lite_path_t * dest, const lv_vector_path_t * src
     LV_PROFILER_END;
 }
 
+static void lv_stroke_to_vg(lv_vg_lite_path_t * dest, const lv_vector_stroke_dsc_t * dsc)
+{
+    LV_ASSERT_NULL(dest);
+    LV_ASSERT_NULL(dsc);
+
+    /* if width is 0, no need to set stroke */
+    if(math_zero(dsc->width)) {
+        return;
+    }
+
+    vg_lite_path_t * path = lv_vg_lite_path_get_path(dest);
+
+    LV_VG_LITE_CHECK_ERROR(vg_lite_set_path_type(path, VG_LITE_DRAW_STROKE_PATH));
+
+    LV_VG_LITE_CHECK_ERROR(
+        vg_lite_set_stroke(
+            path,
+            lv_stroke_cap_to_vg(dsc->cap),
+            lv_stroke_join_to_vg(dsc->join),
+            dsc->width,
+            dsc->miter_limit,
+            lv_array_front(&dsc->dash_pattern),
+            dsc->dash_pattern.size,
+            dsc->width / 2,
+            lv_color32_to_vg(dsc->color, dsc->opa))
+    );
+
+    if(dsc->style == LV_VECTOR_DRAW_STYLE_SOLID) {
```

Comment:
I guess this if-else is here to be extended later. 
Could you please add `// TODO: implement .... ` or remove if-else, if not needed?
In my opinion it is not a problem to place TODOs in source code, later devs can go through them to prioritize which are more important to implement, or can be omitted totally.

Reply:
This judgment has no effect, I will delete it.